.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ commands:
     description: "Commands to run on every Node.js environment"
     steps:
       - run:
-          # rustup must be installed prior to installing `wasm-pack`.
+          # rustup must be installed in order to use `wasm-pack`.
```

Comment:
can we do this in the root repo's `prepare` script instead?

Reply:
Then you really won't have to think about rust if you're just working on the ts code, and we can make the instructions simpler.

Reply:
Is there a way to install Rust entirely hermetically inside the repo? I wouldn't want an npm script to do a global or $HOME-level installation (I think the rustup installer, at least by default, does the latter).

Reply:
Looks like we can set $CARGO_HOME and/or $RUSTUP_HOME (not sure why one vs the other) to install into a directory in the repo. You still need to ensure you set $PATH to include that, of course... and I'm not sure if this will make folks who already have a good Rust setup happy, to have a separate installation?

Reply:
mmm, no i think it would not.

what about throwing the `curl` into a `rustup` npm script and just putting the `which rustup` check into `prepare`, barfing with a message to install `rustup` globally by calling `npm run rustup`?

Reply:
Good call. Appears that `prepare` runs after `postinstall` which is where lerna kicks in, so I put it in postinstall instead.